src/notte/agents/main.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+
+from notte.agents.falco.agent import FalcoAgent, FalcoAgentConfig
+from notte.common.agent.types import AgentResponse
+from notte.common.credential_vault.base import BaseVault
+from notte.common.notifier.base import BaseNotifier, NotifierAgent
+from notte.llms.engine import LlmModel
+from notte.sdk.types import DEFAULT_MAX_NB_STEPS
+
+
+class Agent:
+    def __init__(
+        self,
+        headless: bool = False,
+        reasoning_model: str = LlmModel.default(),  # type: ignore[reportCallInDefaultInitializer]
+        max_steps: int = DEFAULT_MAX_NB_STEPS,
+        use_vision: bool = False,
+        web_security: bool = True,
```

Comment:
also `web_security` is turned on by default, so you might want to exclitely turn it off in the readme to reach the same level of performance